docs/testing-deployment-publishing/branching-collaboration/branching.md

```diff
@@ -0,0 +1,233 @@
+
+1. Create a new branch from the ‘main’ branch.
+
+:::tip
+
+You can also create a new branch from any existing branch.
+
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/5n61rPZR7WuWxs0lTFkE?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+2. Make some changes in the new branch, and from the same menu, select the **Merge** option.
+
+:::info
+
+- You could also select the **Rebase** option, which rewrites the feature branch's history to appear as if it was built directly on top of the latest 'main' branch. However, It's **important to note** that if you're working in a collaborative setting where maintaining an accurate historical record is important, it's safer to simply use the **Merge** option.
+- Know more about Merge vs Rebase [**here**](#merge-vs-rebase).
+
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/VHuZbhTfT8Q8GbaMzMWA?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+3. Now, you'll see a screen that will display if there are any conflicts. If you don’t have any conflicts, you can simply go ahead and click **Merge Branch**.
+
+![merge](../imgs/merge.avif)
+
+### Resolve merge conflicts
+A merge conflict occurs when multiple team members make changes to the same part of the project such as modify the same screen, widget, or configuration simultaneously.
+
+For example, imagine two developers, Alice and Bob, are working on the same FlutterFlow project and both decide to update the same button widget.
+
+**Alice's Changes (Branch: `feature-alice`)**
+
+- Changes the button text to "Submit Form"
+- Changes the button color to blue
+
+**Bob's Changes (Branch: `feature-bob`)**
+
+- Changes the button text to "Send"
+- Changes the button color to green
```

Comment:
Can we make this part into a grid table with branch name and changes as columns? Might look cleaner and easier to explain.

Reply:
Done!